x-pack/plugins/global_search_providers/public/providers/get_app_results.ts

```diff
@@ -40,9 +51,17 @@ export const getAppResults = (
 export const scoreApp = (term: string, appLink: AppLink): number => {
   term = term.toLowerCase();
   const title = [appLink.app.title, ...appLink.subLinkTitles].join(' ').toLowerCase();
+  const appScoreByTerms = scoreAppByTerms(term, title);
+  const appKeywords = appLink.app.meta.keywords.map((keyword) => keyword.toLowerCase());
+  const appSubLinkKeywords = appLink.subLinkKeywords.map((keyword) => keyword.toLowerCase());
+  const keywords = [...appKeywords, ...appSubLinkKeywords];
+  const appScoreByKeywords = scoreAppByKeywords(term, keywords);
+  return Math.max(appScoreByTerms, appScoreByKeywords);
```

Comment:
We probably need to weight the scores towards to Title matches. perhaps the easiest would be some percentage of the total, e.g:
```suggestion
  return Math.max(appScoreByTerms, appScoreByKeywords * 0.8);
```
I'm open to suggestions.

Reply:
A fixed arbitrary ratio seems the easiest solution yea.